src/Action/Api/CreateMollieSubscriptionAction.php

```diff
@@ -0,0 +1,110 @@
+<?php
+declare(strict_types=1);
```

Comment:
No OS header

Reply:
//there is no such file in recurring branch